tests/arrays_tests/includes.test.js

```diff
@@ -0,0 +1,8 @@
+const includes = require('../../tasks/arrays/includes');
+
+test('Returns true if value is found, else false', () => {
+    expect(includes([1, 2, 3], 1)).toEqual(true);
+    expect(includes([1, 2, 3], 1, 2)).toEqual(false);
+    expect(includes({ 'a': 1, 'b': 2 }, 2)).toEqual(true);
+    expect(includes('abcd', 'bc')).toEqual(true);
+});
```

Comment:
test('Returns true if value is found, else false', () => {
  expect(includes([1, 2, 3], 1)).toBe(true);
  expect(includes([1, 2, 3], 1, 2)).toBe(false);
  expect(includes({ 'a': 1, 'b': 2 }, 2)).toBe(true);
  expect(includes('abcd', 'bc')).toBe(true);
});